web/core/src/jsMain/kotlin/androidx/compose/web/css/CSSProperties.kt

```diff
@@ -237,3 +237,70 @@ fun StyleBuilder.padding(value: CSSNumeric) {
     // padding hasn't Typed OM yet
     property("padding", value)
 }
+
+@Suppress("EqualsOrHashCode")
+data class CSSAnimation(
```

Comment:
I see that you usually avoid using `data` classes. 
Is it necessary in this case?

Reply:
It is necessary due to equals check. We discussed with @Schahen that later we will think about replacing data classes with some interface which needs equals to be implemented.